.github/workflows/deployment.yml

```diff
@@ -23,104 +43,144 @@ jobs:
            displayName: macOS
            jpackageDownload: https://download.java.net/java/early_access/jdk14/27/GPL/openjdk-14-ea+27_osx-x64_bin.tar.gz
            jdk14Path: /jdk-14.jdk/Contents/Home
-           archivePortable: tar -czf build/distribution/JabRef-portable_macos.tar.gz -C build/distribution JabRef.app && rm -R build/distribution/JabRef.app
+           archivePortable: brew install pigz && tar -c -C build/distribution JabRef.app | pigz --rsyncable > build/distribution/JabRef-portable_macos.tar.gz && rm -R build/distribution/JabRef.app
```

Comment:
Is this optimization for rsync really necessary? The upload takes like 10s less with it. I would prefer to go with the simple method.

Reply:
I don't know about the implications if we get our archives right. I am in the middle of working on the data transfer to have a data transfer happen only if the file contents "really" changed.